.github/workflows/ci-backend.yml

```diff
@@ -87,9 +87,11 @@ jobs:
             - uses: actions/checkout@v3
               with:
                   fetch-depth: 1
+                  path: 'current/'
```

Comment:
why does it need to be in a directory now?

Reply:
The antlr test below downloads antlr into a directory and then uses `git diff` to check for changes. Downloading it into the repo always guarantees changes and a failing test... so I had to move them all to folders.